src/lib.rs

```diff
@@ -77,12 +77,8 @@ pub async fn run_trin(
     )?;
 
     // Initialize validation oracle
-    let pre_merge_accumulator =
-        PreMergeAccumulator::try_from_file(trin_config.pre_merge_acc_path.clone())?;
-    info!(
-        "Loaded pre-merge accumulator from: {:?}",
-        trin_config.pre_merge_acc_path
-    );
+    let pre_merge_accumulator = PreMergeAccumulator::default();
+    info!("Loaded pre-merge accumulator.",);
```

Comment:
It seems worthwhile putting the accumulator hash in this message, then users can confirm the hash matches against an external / 3rd party source if they want.

Reply:
Forgot to address this before merging, will add it to my next PR.